### PR TITLE
 [web_timeline][IMP] Add margin support in web_timeline

### PR DESCRIPTION
**Pull request by cvinh**
According to the version number in manifest file, this PR should be merged after https://github.com/OCA/web/pull/1134

### CODE REVIEW

File: web_timeline/README.rst [hunk 1]
```diff
@@ -56,6 +56,9 @@ the possible attributes for the tag:
...
 | mode               | No        | Specifies the initial visible window. Available values are: 'day' to display the current day, 'week', 'month' and 'fit'. Default value is 'fit' to adjust the visible window such that it fits all items.                                                                 |
 +--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| margin             | No        | Specifies the margins around the items. It should respect the JSON format. For example '{"item":{"horizontal":-10}}'. Available values are: '{"axis":<number>}', '{"item":<number>}', '{"item":{"horizontal":<number>}}', '{"item":{"vertical":<number>}}',               |
```
Comment (pedrobaeza) [inline web_timeline/README.rst]: This line seems bad formatted. Maybe too long?

**Comment by pedrobaeza** [comment]
Please rebase this one and fix conflicts + the problem in the README

File: web_timeline/README.rst (hunk 1 above)
Comment (tarteo) [inline web_timeline/README.rst]: Can you mention that the options can be combined. Also the documentation of visjs says something different then here.

> The minimal margin in pixels between items and the time axis.

And 

> The minimal margin in pixels between items in both horizontal and vertical direction.

Can you change it to this?

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks. Can you fix the conflicts, please.

**Comment by pedrobaeza** [comment]
There are still conflicts. You have to rebase the branch over the new origin.

**Comment by cvinh** [comment]
> There are still conflicts. You have to rebase the branch over the new origin.

I think it's clean now... could you check it out ?

**Review by pedrobaeza — APPROVED** [review]
To be squashed on merge.

**Review by tarteo — APPROVED** [review]

**Pull request merged**